### PR TITLE
docs: add test account credentials to README

### PR DESCRIPTION
**Pull request by shivamgupta2020**
Added test account credentials in README for ease in login in dev mode.

### CODE REVIEW

**Comment by thevaibhav-dixit** [comment]
pls follow conventional commit specification

File: README.md [hunk 1]
```diff
@@ -178,6 +178,11 @@ It will also remove the containers and, consequentially, the data held in them.
 buck2 run dev:down
 ```
 
+### (8) Test account credentials
```
Comment (openoms) [inline README.md:181]: Suggest to make this step 5 after `Running the Stack` .

Comment (shivamgupta2020) [reply README.md]: updated in next commit

**Comment by shivamgupta2020** [comment]
> pls follow conventional commit specification

I will make sure to follow this in future.

File: README.md [hunk 2]
```diff
@@ -165,6 +165,11 @@ buck2 run dev:up
 Once Tilt starts, you can check on the status of all services by accessing the UI through the given port on your local host (e.g. [http://localhost:10350/](http://localhost:10350/)).
 Every service should eventually have a green checkmark next to them, which ensures that they are in "ready" states.
 
+
+### (5) Test account credentials
+
+For account login, you can input any random mobile number/email and then use `000000` as the OTP.
```
Comment (nicolasburtey) [inline README.md:171]:
```suggestion
For account login on regtest, you can input any random mobile number and then use `000000` as the OTP.
```

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**